fontforge/python.c

```diff
@@ -1241,12 +1243,62 @@ static PyObject *PyFF_scriptFromUnicode(PyObject *UNUSED(self), PyObject *args)
     return TagToPythonString(script, false);
 }
 
+static bool sanitize_str_for_printf(char** message, const char* warning_fmt) {
```

Comment:
I wasn't sure where to put this, there wasn't some clear 'string utils' place, so I placed it in python.c before it's used.

Let me know where and I can move it.